### PR TITLE
Capstone Project

### PR DESCRIPTION
**Pull request by phelian23**
In this PR
- React-redux was set up
- Linters was installed
- API was fetched
- Homepage was added
- Details page was added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,53 @@
...
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- Redux
+
+## Application Page url
+
+[Live Demo](https://hopeful-easley-4dce01.netlify.app/)
```
Comment (omar-labana) [inline README.md]: - [ ] The link to the live version is broken, please make sure to correctly deploy your app.

File: src/components/allSports.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      <div className={styles.subMain}>
+        <ul className={styles.list}>
+          {categories.map((coin) => (
+            <Link key={coin.name} className={styles.link} to="/sports" onClick={() => dispatch(displayLeague(coin.name))}>
+              <li className={styles.listItem}>
+                <h3 className={styles.listName}>{coin.name}</h3>
+                <p>{sumCap(coin.coinArr)}</p>
+              </li>
+            </Link>
+          ))}
```
Comment (omar-labana) [inline src/components/allSports.js]: - [ ] Each of the category items detail page should have its unique route within the SPA (currently they all have the same `/sports` pathname). I'd suggest using dynamic routing, it should look like this `<Route exact path="/item/:id" component={Item} />`.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @phelian23, 

Excellent work so far! 👌
You correctly implemented all of the required technical features. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️

- [ ] Kindly, add more tests to cover unit tests for several components of the app and some pure functions in Redux code. Also, it would be nice to add some integration tests using React Testing Library.
- [ ] _[NOTE]_ If you take a look at the branch name it's not a name suitable for this submission, kindly change it to something that relates to this submission.
- [ ] _Please, check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/homepage.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import AllSports from './allSports';
+import Sport from './sport';
+
+const HomePage = () => (
+  <div>
+    <Routes>
+      {['/', '/react-redux-capstone'].map((path) => <Route path={path} element={<AllSports />} key="key-1" />)}
+      <Route path="/*" element={<Sport />} />
```
Comment (rukundoeric) [inline src/components/homepage.js]: - [ ] <b>REQUIRED:</b>  You have two routes `['/', '/react-redux-capstone']` that are rendering the same component `<AllSports />`.  the project requires to have at least one route to render the list of items and another one to render the details of a single item. Besides, the is no point in having two routes rendering the same component when there is no `redirect` functionality.  Kindly address it and fix it.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @phelian23, 

_Great job so far 👏 💯 👍. You've done well implementing some of the requested changes. However, there are still some which aren't addressed yet. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 
  - Addressed some of the issues raised from the previous review. 💯 

## Required Changes ♻️
  - [ ] There are still a lot of complications in your code base. 
      - Some components are called `sport or allsports` when there are actually rendering Cryptocurrencies. So kindly refactor your code and make sure to remove those complications.
      - You have two routes ['/', '/react-redux-capstone'] that are rendering the same component <AllSports />. the project requires to have at least one route to render the list of items and another one to render the details of a single item.
  - [ ]  As the previous reviewer mentioned, you still haven't implemented `filter` functionality(by category or any other thing). This is a mandatory feature, Kindly implement it.
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
  
  - N/A 

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**